sys/shell/commands/sc_gnrc_ipv6_nib.c

```diff
@@ -114,12 +114,13 @@ static int _nib_neigh(int argc, char **argv)
     }
     else if ((argc > 3) && (strcmp(argv[2], "del") == 0)) {
         ipv6_addr_t ipv6_addr;
+        unsigned iface = atoi(argv[3])
 
-        if (ipv6_addr_from_str(&ipv6_addr, argv[3]) == NULL) {
+        if (ipv6_addr_from_str(&ipv6_addr, argv[4]) == NULL) {
```

Comment:
please adapt help/usage for `nib neigh` accordingly, i.e. document the new iface parameter,

Reply:
Done